package.json

```diff
@@ -26,7 +26,7 @@
         "url": "https://github.com/dracula/visual-studio-code/issues"
     },
     "engines": {
-        "vscode": "^1.12.0"
+        "vscode": "^1.12.1"
```

Comment:
This should stay at `^1.12.0`. I don't think that last patch they made affected the UI scopes.